### PR TITLE
Mac: Fix keyboard shortcut for adding new carets

### PR DESCRIPTION
**Pull request by sandyarmstrong**
The keys to hold while clicking to add new carets are control+option, not command+option.

### CODE REVIEW

**Comment by PRMerger18** [comment]
@sandyarmstrong : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by jmatthiesen — APPROVED** [review]
Great catch, thanks for the update! #sign-off

**Comment by ShannonLeavitt** [comment]
@jmatthiesen - Thanks for approving the PR. For future PRs, please note that a `#sign-off` comment is the best way to signal to the review team that a PR is ready to merge.

Because HTML is no longer allowed in Docs, I will replace the HTML with capitalization, per the style guide. Then I'll merge the PR.

File: mac/multi-caret-editing.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Each of these behaviors are anchored to the current position of the caret when y
...
+To use the mouse to insert multiple carets, you must press and hold <kbd>control</kbd> and <kbd>option</kbd> and click where you would like the carets to be entered. You will be in insertion mode as long as the <kbd>control</kbd> and <kbd>option</kbd> keys are held. If you insert a caret in an incorrect location, you can remove the caret by continuing to hold <kbd>control</kbd> and <kbd>option</kbd> and clicking in the same area again. Once you have all of the carets located where you would like them, stop pressing the <kbd>control</kbd> and <kbd>option</kbd> keys and start typing. The following GIF demonstrates both selecting a set of insertion points as well as removing erroneously set points.
```
Comment (ShannonLeavitt) [inline mac/multi-caret-editing.md:35]:
```suggestion
To use the mouse to insert multiple carets, you must press and hold the Control key and Option key and click where you would like the carets to be entered. You will be in insertion mode as long as the Control and Option keys are held. If you insert a caret in an incorrect location, you can remove the caret by continuing to hold the Control key and Option key and clicking in the same area again. Once you have all of the carets located where you would like them, stop pressing the Control and Option keys and start typing. The following GIF demonstrates both selecting a set of insertion points as well as removing erroneously set points.
```
Removing HTML

**Comment by sandyarmstrong** [comment]
Thanks, I wasn't sure of the right style for shortcut keys these days.

**Comment by PRMerger17** [comment]
@sandyarmstrong : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**